cl_dll/vgui_TeamFortressViewport.cpp

```diff
@@ -1429,6 +1447,14 @@ void TeamFortressViewport::HideScoreBoard( void )
 	if ( gHUD.m_iIntermission )
 		return;
 
+	// The user can change the scoreboard style while the scoreboard is being "drawn" (e.g. when togglescores is on)
+	// Therefore hide both of them so that we are sure we aren't drawing both of them
+
+	if ( CVAR_GET_FLOAT("cl_old_scoreboard") == 1 )
```

Comment:
Um, the function starts with
```c++
void TeamFortressViewport::HideScoreBoard( void )
{
	// Prevent removal of scoreboard during intermission
	if ( gHUD.m_iIntermission )
		return;
```
Why should it hide during an intermission?

Reply:
Oh right, I am smart. I guess you mean after the intermission is over... Well, it "doesn't hide", but it won't show up either, because after an intermission the map changes and VidInit() is called again?

Reply:
Anyhow, removed the check altogether.